src/utilities/duration.ts

```diff
@@ -0,0 +1,67 @@
+const MINUTE = 60;
```

Comment:
Use this file from polaris
`@polaris/utilities/duration.ts`
All ts files that does not include React import inside, we can use directly.